### PR TITLE
Hali2/sdkupdate

### PR DESCRIPTION
**Pull request by HanLiMS**

### CODE REVIEW

File: src/SDKs/CognitiveServices/dataPlane/Language/TextAnalytics/TextAnalytics.Tests/Microsoft.Azure.CognitiveServices.Language.TextAnalytics.Tests.csproj [hunk 1]
```diff
@@ -4,11 +4,18 @@
     <Description>Microsoft.Azure.CognitiveServices.Language.TextAnalytics.Tests Class Library</Description>
     <AssemblyName>Microsoft.Azure.CognitiveServices.Language.TextAnalytics.Tests</AssemblyName>
     <Version>1.0.0</Version>    
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>    
+    <Platforms>AnyCPU;x64;x86</Platforms>
```
Comment (dsgouda) [inline src/SDKs/CognitiveServices/dataPlane/Language/TextAnalytics/TextAnalytics.Tests/Microsoft.Azure.CognitiveServices.Language.TextAnalytics.Tests.csproj]: Please remove this line

File: src/SDKs/CognitiveServices/dataPlane/Language/TextAnalytics/TextAnalytics.Tests/Microsoft.Azure.CognitiveServices.Language.TextAnalytics.Tests.csproj [hunk 2]
```diff
@@ -4,11 +4,18 @@
...
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Management.ResourceManager" Version="1.6.0-preview" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.9.0" />
+    <PackageReference Include="xunit" Version="2.4.1" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.1">
+      <PrivateAssets>all</PrivateAssets>
```
Comment (dsgouda) [inline src/SDKs/CognitiveServices/dataPlane/Language/TextAnalytics/TextAnalytics.Tests/Microsoft.Azure.CognitiveServices.Language.TextAnalytics.Tests.csproj]: You don't need these lines

File: src/SDKs/CognitiveServices/dataPlane/Language/TextAnalytics/TextAnalytics.sln [hunk 3]
```diff
@@ -10,17 +10,37 @@ EndProject
...
+		Debug|x64 = Debug|x64
+		Debug|x86 = Debug|x86
 		Release|Any CPU = Release|Any CPU
+		Release|x64 = Release|x64
+		Release|x86 = Release|x86
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
 		{6807B854-8528-4FEE-A25D-C43C3AA2D601}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{6807B854-8528-4FEE-A25D-C43C3AA2D601}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{6807B854-8528-4FEE-A25D-C43C3AA2D601}.Debug|x64.ActiveCfg = Debug|x64
```
Comment (dsgouda) [inline src/SDKs/CognitiveServices/dataPlane/Language/TextAnalytics/TextAnalytics.sln]: Were these changes intended?

File: src/SDKs/CognitiveServices/dataPlane/Language/TextAnalytics/TextAnalytics/Microsoft.Azure.CognitiveServices.Language.TextAnalytics.csproj [hunk 4]
```diff
@@ -6,7 +6,7 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.CognitiveServices.Language.TextAnalytics</PackageId>
     <Description>This client library provides access to the Microsoft Cognitive Services Language APIs.</Description>
-    <Version>3.0.0</Version>
+    <Version>3.0.1</Version>
     <AssemblyName>Microsoft.Azure.CognitiveServices.Language.TextAnalytics</AssemblyName>
     <PackageTags>Microsoft Cognitive Services;Cognitive Services;Cognitive Services SDK;Text Analytics API;Text Analytics;REST HTTP client;netcore451511</PackageTags>
     <PackageReleaseNotes>
```
Comment (dsgouda) [inline src/SDKs/CognitiveServices/dataPlane/Language/TextAnalytics/TextAnalytics/Microsoft.Azure.CognitiveServices.Language.TextAnalytics.csproj]: Please update `PackageReleaseNotes`

File: src/SDKs/CognitiveServices/dataPlane/Language/TextAnalytics/TextAnalytics/Microsoft.Azure.CognitiveServices.Language.TextAnalytics.csproj [hunk 5]
```diff
@@ -19,10 +19,14 @@
     1. Add support for v2.1 API version which includes Entity Type and Wikipedia confidence score to the /Entities endpoint.
     
     ]]></PackageReleaseNotes>
+    <Platforms>AnyCPU;x64;x86</Platforms>
   </PropertyGroup>
   <PropertyGroup>
     <TargetFrameworks>$(SdkTargetFx)</TargetFrameworks>
   </PropertyGroup>
+  <PropertyGroup>
```
Comment (dsgouda) [inline src/SDKs/CognitiveServices/dataPlane/Language/TextAnalytics/TextAnalytics/Microsoft.Azure.CognitiveServices.Language.TextAnalytics.csproj]: Please remove unnecessary changes

File: src/SDKs/CognitiveServices/dataPlane/Language/TextAnalytics/TextAnalytics/Microsoft.Azure.CognitiveServices.Language.TextAnalytics.csproj [hunk 6]
```diff
@@ -19,10 +19,14 @@
     1. Add support for v2.1 API version which includes Entity Type and Wikipedia confidence score to the /Entities endpoint.
     
     ]]></PackageReleaseNotes>
+    <Platforms>AnyCPU;x64;x86</Platforms>
```
Comment (dsgouda) [inline src/SDKs/CognitiveServices/dataPlane/Language/TextAnalytics/TextAnalytics/Microsoft.Azure.CognitiveServices.Language.TextAnalytics.csproj]: Please remove unnecessary changes

File: src/SDKs/CognitiveServices/dataPlane/Language/TextAnalytics/TextAnalytics/Generated/TextAnalytics/TextAnalyticsClientExtensions.cs [hunk 7]
```diff
@@ -135,13 +136,172 @@ public static async Task<KeyPhraseBatchResult> KeyPhrasesAsync(this ITextAnalyti
             /// <param name='cancellationToken'>
             /// The cancellation token.
             /// </param>
-            public static async Task<SentimentBatchResult> SentimentAsync(this ITextAnalyticsClient operations, bool? showStats = default(bool?), MultiLanguageBatchInput multiLanguageBatchInput = default(MultiLanguageBatchInput), CancellationToken cancellationToken = default(CancellationToken))
```
Comment (dsgouda) [inline src/SDKs/CognitiveServices/dataPlane/Language/TextAnalytics/TextAnalytics/Generated/TextAnalytics/TextAnalyticsClientExtensions.cs]: This is a breaking change and needs a major version bump for the nuget package.

**Review by dsgouda — CHANGES_REQUESTED** [review]
How was this code generated? Please use the generate.ps1 script to generate the code.
Also, please bump the version numbers in AssemblyInfo.cs

**Comment by dsgouda** [comment]
Please join the Azure org [here](https://aka.ms/azuregithub) to kick off the Jenkins Validation

**Comment by dsgouda** [comment]
@HanLiMS gentle ping. PR will be closed if inactive by EOW

**Comment by dsgouda** [comment]
@HanLiMS any updates here, we will close this PR if inactive by EOW. Please open a new PR when ready

**Comment by assafi** [comment]
> @HanLiMS any updates here, we will close this PR if inactive by EOW. Please open a new PR when ready

@dsgouda - Thanks for the comments. Han has been OOF lately, we'll take over. Please don't close this PR.

File: src/SDKs/CognitiveServices/dataPlane/Language/TextAnalytics/TextAnalytics.sln (hunk 3 above)
Comment (HanLiMS) [reply src/SDKs/CognitiveServices/dataPlane/Language/TextAnalytics/TextAnalytics.sln]: No. Updated.

File: src/SDKs/CognitiveServices/dataPlane/Language/TextAnalytics/TextAnalytics.Tests/Microsoft.Azure.CognitiveServices.Language.TextAnalytics.Tests.csproj (hunk 2 above)
Comment (HanLiMS) [reply src/SDKs/CognitiveServices/dataPlane/Language/TextAnalytics/TextAnalytics.Tests/Microsoft.Azure.CognitiveServices.Language.TextAnalytics.Tests.csproj]: OK. Updated.

File: src/SDKs/CognitiveServices/dataPlane/Language/TextAnalytics/TextAnalytics.Tests/Microsoft.Azure.CognitiveServices.Language.TextAnalytics.Tests.csproj (hunk 1 above)
Comment (HanLiMS) [reply src/SDKs/CognitiveServices/dataPlane/Language/TextAnalytics/TextAnalytics.Tests/Microsoft.Azure.CognitiveServices.Language.TextAnalytics.Tests.csproj]: Removed

File: src/SDKs/CognitiveServices/dataPlane/Language/TextAnalytics/TextAnalytics/Microsoft.Azure.CognitiveServices.Language.TextAnalytics.csproj (hunk 4 above)
Comment (HanLiMS) [reply src/SDKs/CognitiveServices/dataPlane/Language/TextAnalytics/TextAnalytics/Microsoft.Azure.CognitiveServices.Language.TextAnalytics.csproj]: Updated

File: src/SDKs/CognitiveServices/dataPlane/Language/TextAnalytics/TextAnalytics/Microsoft.Azure.CognitiveServices.Language.TextAnalytics.csproj (hunk 5 above)
Comment (HanLiMS) [reply src/SDKs/CognitiveServices/dataPlane/Language/TextAnalytics/TextAnalytics/Microsoft.Azure.CognitiveServices.Language.TextAnalytics.csproj]: Removed

File: src/SDKs/CognitiveServices/dataPlane/Language/TextAnalytics/TextAnalytics/Microsoft.Azure.CognitiveServices.Language.TextAnalytics.csproj (hunk 6 above)
Comment (HanLiMS) [reply src/SDKs/CognitiveServices/dataPlane/Language/TextAnalytics/TextAnalytics/Microsoft.Azure.CognitiveServices.Language.TextAnalytics.csproj]: Removed

File: src/SDKs/CognitiveServices/dataPlane/Language/TextAnalytics/TextAnalytics/Generated/TextAnalytics/TextAnalyticsClientExtensions.cs (hunk 7 above)
Comment (HanLiMS) [reply src/SDKs/CognitiveServices/dataPlane/Language/TextAnalytics/TextAnalytics/Generated/TextAnalytics/TextAnalyticsClientExtensions.cs]: There is no major change in Text Analytics API. We would like to avoid auto-generate and directly update the sdk here. Should we use version number <3.1.0>?

**Comment by dsgouda** [comment]
@HanLiMS Please join the Azure org as instructed

File: src/SDKs/CognitiveServices/dataPlane/Language/TextAnalytics/TextAnalytics.Tests/Microsoft.Azure.CognitiveServices.Language.TextAnalytics.Tests.csproj [hunk 8]
```diff
@@ -3,8 +3,7 @@
   <PropertyGroup>
     <Description>Microsoft.Azure.CognitiveServices.Language.TextAnalytics.Tests Class Library</Description>
     <AssemblyName>Microsoft.Azure.CognitiveServices.Language.TextAnalytics.Tests</AssemblyName>
-    <Version>1.0.0</Version>    
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
```
Comment (dsgouda) [inline src/SDKs/CognitiveServices/dataPlane/Language/TextAnalytics/TextAnalytics.Tests/Microsoft.Azure.CognitiveServices.Language.TextAnalytics.Tests.csproj]: Please undo all the changes to this file.

**Review by dsgouda — CHANGES_REQUESTED** [review]
Looks good for the most part

File: src/SDKs/CognitiveServices/dataPlane/Language/TextAnalytics/TextAnalytics.Tests/Microsoft.Azure.CognitiveServices.Language.TextAnalytics.Tests.csproj (hunk 8 above)
Comment (HanLiMS) [reply src/SDKs/CognitiveServices/dataPlane/Language/TextAnalytics/TextAnalytics.Tests/Microsoft.Azure.CognitiveServices.Language.TextAnalytics.Tests.csproj]: Updated for this file.

**Review by dsgouda — APPROVED** [review]
LGTM

**Review by dsgouda — CHANGES_REQUESTED** [review]
Please generate the code using the [generate.ps1](https://github.com/Azure/azure-sdk-for-net/blob/233f3be886fd830e0dd09a8213a2bdff113ea17d/src/SDKs/CognitiveServices/dataPlane/Language/TextAnalytics/TextAnalytics/generate.ps1) script and update the PR

File: src/SDKs/CognitiveServices/dataPlane/Language/TextAnalytics/TextAnalytics/Generated/TextAnalytics/Models/LanguageInput.cs [hunk 9]
```diff
@@ -27,7 +27,7 @@ public LanguageInput()
         /// Initializes a new instance of the LanguageInput class.
         /// </summary>
         /// <param name="id">Unique, non-empty document identifier.</param>
-        public LanguageInput(string countryHint = default(string), string id = default(string), string text = default(string))
```
Comment (dsgouda) [inline src/SDKs/CognitiveServices/dataPlane/Language/TextAnalytics/TextAnalytics/Generated/TextAnalytics/Models/LanguageInput.cs]: How are these changes being generated?

Comment (dsgouda) [reply src/SDKs/CognitiveServices/dataPlane/Language/TextAnalytics/TextAnalytics/Generated/TextAnalytics/Models/LanguageInput.cs]: @HanLiMS please take a look

**Comment by dsgouda** [comment]
@HanLiMS PR will be closed by EOW if there is no activity here

File: src/SDKs/CognitiveServices/dataPlane/Language/TextAnalytics/TextAnalytics/Generated/TextAnalytics/TextAnalyticsClientExtensions.cs (hunk 7 above)
Comment (assafi) [reply src/SDKs/CognitiveServices/dataPlane/Language/TextAnalytics/TextAnalytics/Generated/TextAnalytics/TextAnalyticsClientExtensions.cs]: Opt for 3.1.0 version bump

**Comment by assafi** [comment]
@dsgouda I have opened a new [PR](https://github.com/Azure/azure-sdk-for-net/pull/6291) which supersedes this one. It is based off this PR and includes the Customizations updates you have requested. 
Feel free to close this PR.

**Comment by dsgouda** [comment]
@HanLiMS  Closing this PR due to upcoming directory restructuring changes. Please open the PR manually after 06/11 when the restructuring is complete.
